R/review_structures.R

```diff
@@ -253,7 +280,8 @@ RS_compute_delta_memory <- function(state, df){
   id_hashes <- RS_compute_id_hashes(df, id_vars) |> c() |> array(dim = c(16L, nrow(df)))
 
   tracked_vars <- state$tracked_vars
-  tracked_hashes <- (apply(df[tracked_vars], 1, SH$hash_tracked, simplify = TRUE) |> c() |> 
+  # FIXME: (LUIS): Ask Miguel about the postlude
+  tracked_hashes <- (SH$hash_tracke(df[tracked_vars]) |> c() |> 
```

Comment:
We can discuss the "postlude" thing live.

There's some partial matching here that we could avoid:
```suggestion
  tracked_hashes <- (SH$hash_tracked(df[tracked_vars]) |> c() |> 
```